src/patternfly/components/Masthead/examples/masthead.md

```diff
@@ -0,0 +1,140 @@
+---
+id: 'Masthead'
+beta: true
+section: components
+cssPrefix: pf-c-masthead
+---
+
+## Examples
+### Basic
+```hbs
+{{#> masthead masthead--id="basic-masthead"}}
+  {{> masthead-toggle}}
+  {{#> masthead-main}}
+    {{#> masthead-brand}}
+      Logo
+    {{/masthead-brand}}
+  {{/masthead-main}}
+  {{> masthead-content}}
+{{/masthead}}
+```
+
+### Display inline
+```hbs
+{{#> masthead masthead--id="inline-masthead" masthead--modifier="pf-m-display-inline"}}
+  {{> masthead-toggle}}
+  {{#> masthead-main}}
+    {{#> masthead-brand}}
+      Logo
+    {{/masthead-brand}}
+  {{/masthead-main}}
+  {{> masthead-content}}
+{{/masthead}}
+```
+
+### Display stack
+```hbs
+{{#> masthead masthead--id="stack-masthead" masthead--modifier="pf-m-display-stack"}}
+  {{> masthead-toggle}}
+  {{#> masthead-main}}
+    {{#> masthead-brand}}
+      Logo
+    {{/masthead-brand}}
+  {{/masthead-main}}
+  {{> masthead-content}}
+{{/masthead}}
+```
+
+### Display stack, display inline responsive
+```hbs
+{{#> masthead masthead--id="stack-inline-masthead" masthead--modifier="pf-m-display-inline pf-m-display-stack-on-lg pf-m-display-inline-on-2xl"}}
+  {{> masthead-toggle}}
+  {{#> masthead-main}}
+    {{#> masthead-brand}}
+      Logo
+    {{/masthead-brand}}
+  {{/masthead-main}}
+  {{> masthead-content}}
+{{/masthead}}
+```
+
+### Light variant
```

Comment:
Should this title be "Light 100 variant"? Or maybe we could rename the modifier to `pf-m-light` to align with the page section variation? Or just leave it as it is?